src/__tests__/pages/policy/input/ParameterEditor.test.js

```diff
@@ -0,0 +1,61 @@
+// src/components/ParameterEditor.test.jsx
+
+import React from "react";
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import ParameterEditor from "../../../../pages/policy/input/ParameterEditor";
+import { BrowserRouter } from "react-router-dom";
+
+global.URL.createObjectURL = jest.fn();
```

Comment:
Could you elaborate on the purpose of this line?

Reply:
Would also be great to have a comment in the test saying what its purpose is

Reply:
Understood, at first due to some reason I was facing an error saying createObjectURL is undefined or something like that, so I mocked it and it passed. But now there's no error even after removing this line, so I removed it in the current changes. I have also added comments in the test code